### ISSUE TITLE
Tutorial step: "Let's Add your Review" broken

### ISSUE THREAD

**Issue by potens1** [Superalgos/Superalgos#1193]
## Before you open an issue

There are two requirements to open an issue:

1. You must first follow the [Troubleshooting Guide](https://docs.superalgos.org/suite-how-to-troubleshoot-an-issue.html)

2. If your issue hasn't been resolved by following the Troubleshooting Guide, then follow the guide on [How to Report an Issue](https://docs.superalgos.org/suite-how-to-report-an-issue.html).

## Context

*Complete and adjust the contents leaving the information that fits your case and deleting the information that doesn't.*

**Operating System:** Linux Arch

**Software version:** today's Plugins-Doc branch ( 609806f475a022b10b4e2580d7b102bee0401b98 )

**Workspace version:** default Workspace.json (Getting Started Guide) 

**Reproducible:** the issue may be reproduced.

### 1. Explain what the problem seems to be.

Unable to run the docs.add command

### 2. Clearly explain what you wish to accomplish.

Fulfill the tutorial Beyond the Basics -> Editing -> Let's Add your review

### 3. Explain, step by step, what you've done to achieve the goal. The Team will follow these steps to reproduce the issue, so be precise.

I type `docs.add Review to Superalgos: Superalgos Project Review->Superalgos Review - Tremendous` into the command text field

### 4. Explain what is the result you expect from your actions.

I guess create a new page

### 5. Explain what happens instead of the expected results.

Nothing

### 6. Explain what errors you have found on-screen, on the browser console application, or in the ```Log-Files``` folder.
As soon as the UI starts (often repeated) :
```
[Error] Could not get APP Scchema for project Superalgos
```
Those are the schemas I can see in console:
```
> SCHEMAS_BY_PROJECT
< Map(5) {"Foundations" => {…}, "Ethereum" => {…}, "TensorFlow" => {…}, "Governance" => {…}, "Education" => {…}}
[[Entries]]
0: {"Foundations" => Object}
1: {"Ethereum" => Object}
2: {"TensorFlow" => Object}
3: {"Governance" => Object}
4: {"Education" => Object}

> ﻿﻿﻿PROJECTS_SCHEMA
< (5) [{…}, {…}, {…}, {…}, {…}]
0:
CL: {}
TS: {globals: Array(10), functionLibraries: Array(9), utilities: Array(4), taskModules: Array(3), processModules: Array(6), …}
UI: {globals: Array(1), functionLibraries: Array(22), utilities: Array(19), spaces: Array(8)}
name: "Foundations"
plugins: (9) ["Data Mines", "Trading Mines", "Learning Mines", "Trading Systems", "Learning Systems", "Trading Engines", "Learning Engines", "Tutorials", "API Maps"]
propertyName: "foundations"
__proto__: Object
1: {name: "Ethereum", propertyName: "ethereum", UI: {…}, TS: {…}, CL: {…}}
2: {name: "TensorFlow", propertyName: "tensorflow", UI: {…}, TS: {…}}
3: {name: "Governance", propertyName: "governance", UI: {…}, TS: {…}, CL: {…}, …}
4: {name: "Education", propertyName: "education", UI: {…}, TS: {…}, CL: {…}}
length: 5
__proto__: Array(0)
(5) [{…}, {…}, {…}, {…}, {…}]

```

When trying to run the command:
```
Docs.js:44 Uncaught TypeError: Cannot read property 'array' of undefined
    at Object.buildOrderedPageIndex (Docs.js:44)
    at addReview (CommandInterface.js:803)
    at checkAddCommand (CommandInterface.js:291)
    at detectDocsCommands (CommandInterface.js:139)
    at Object.detectCommands (CommandInterface.js:21)
    at HTMLInputElement.<anonymous> (MainSearchPage.js:59)
```
Process log (unrelated, I guess)

```
[ERROR] httpInterface -> respondWithImage -> onFileRead -> File Not Found: ./Projects/Foundations/Icons/tutorial.png or Error = Error: ENOENT: no such file or directory, open './Projects/Foundations/Icons/tutorial.png'
[ERROR] httpInterface -> respondWithImage -> onFileRead -> File Not Found: ./Projects/Foundations/Icons/strategy-manage-stage.png or Error = Error: ENOENT: no such file or directory, open './Projects/Foundations/Icons/strategy-manage-stage.png'
[ERROR] httpInterface -> respondWithImage -> onFileRead -> File Not Found: ./Projects/Foundations/Icons/nodes-highlights.png or Error = Error: ENOENT: no such file or directory, open './Projects/Foundations/Icons/nodes-highlights.png'
[ERROR] httpInterface -> respondWithImage -> onFileRead -> File Not Found: ./Projects/Foundations/Icons/bullet-error.png or Error = Error: ENOENT: no such file or directory, open './Projects/Foundations/Icons/bullet-error.png'
[ERROR] httpInterface -> respondWithImage -> onFileRead -> File Not Found: ./Projects/Foundations/Icons/bullet-warning.png or Error = Error: ENOENT: no such file or directory, open './Projects/Foundations/Icons/bullet-warning.png'
[ERROR] httpInterface -> respondWithImage -> onFileRead -> File Not Found: ./Projects/Foundations/Icons/docs-success-style.png or Error = Error: ENOENT: no such file or directory, open './Projects/Foundations/Icons/docs-success-style.png'
[ERROR] httpInterface -> respondWithImage -> onFileRead -> File Not Found: ./Projects/Foundations/Icons/docs-action-edit-paragraph.png or Error = Error: ENOENT: no such file or directory, open './Projects/Foundations/Icons/docs-action-edit-paragraph.png'
[ERROR] httpInterface -> respondWithImage -> onFileRead -> File Not Found: ./Projects/Foundations/Icons/dataset.png or Error = Error: ENOENT: no such file or directory, open './Projects/Foundations/Icons/dataset.png'

```

## Attach relevant log files and error screenshots

![image](https://user-images.githubusercontent.com/3988571/122904304-7fce4100-d350-11eb-909b-4de862573d3f.png)

**Comment by harrellbm** [comment]
This section needs updated in the docs.  The file structure just got changed and refactored.

The command you should type is "docs.add Review to Foundations: Superalgos Project Review->Superalgos Review - Tremendous"

**Comment by SingingDwarf** [comment]
This was fixed in commit 2d91c1c on 2 Jul in develop
